src/content/doc-cloud/advanced-topics/monitoring/metrics.mdx

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 3
+sidebar_label: Metrics
+title: Metrics
+description: Learn how to monitor the metrics of your Surreal Cloud Instance. 
+---
+
+import Image from "@components/Image.astro";
+import LightMetrics from "@img/image/cloud/metrics.png";
+import DarkMetrics from "@img/image/cloud/metrics.png";
+
+# Metrics
+
+In this section you will explore the different metrics that are available for your Surreal Cloud Instance. The following metrics are available for your Surreal Cloud Instance:
+
+- **System**: this includes the compute and memory usage of your Surreal Cloud Instance. 
+- **Connections**: this includes HTTP and RPC requests to your Surreal Cloud Instance. 
+- **Network traffic**: this includes the network egress and ingress traffic to and from your Surreal Cloud Instance. 
+
+To access the metrics of your Surreal Cloud Instance, navigate to the **monitoring** tab in the sidebar. Under the **metrics** option, you will see a list of metrics for your Surreal Cloud Instance. 
+
+<Image
+alt="Configure an Instance"
+src={{
+	light: LightMetrics,
+	dark: DarkMetrics,
+	}}  
+/>
+
+You will see a list of metrics for your Surreal Cloud Instance. 
+
+### Filtering metrics
+
+While viewing the metrics of your Surreal Cloud Instance, you can filter the metrics by selecting the **filter** button. This will open a modal where you can select the metrics you want to view. You can filter by the last hour, last 12 hours, last day last week and last month. 
```

Comment:
```suggestion
While viewing the metrics of your Surreal Cloud Instance, you can filter the metrics by selecting the **filter** button. This will open a modal where you can select the metrics you want to view. You can filter by the last hour, last 12 hours, last day, last week, and last month. 
```